src/Illuminate/Support/Facades/Log.php

```diff
@@ -5,6 +5,16 @@
 use Psr\Log\LoggerInterface;
 
 /**
+ * @method static void emergency(string $message, array $context = array())
```

Comment:
Why is the long array syntax used here, but then the short array syntax used in other places, e.g. https://github.com/laravel/framework/pull/23694/files#diff-335842ccd61aadfa0c1ada80c98bb179R8?

Reply:
@martinbean , 
I have copied from the Contracts.
I agree then it is not so good... but as for me will be better to save the original syntactic from the method.

Reply:
It's better to change it to short array syntax `[]` because it breaks code styling.

Reply:
@a-komarev , ok.
I will create a pull request later.

Reply:
@a-komarev , @martinbean , https://github.com/laravel/framework/pull/23707